assets/_scss/default/body-main-styles.scss

```diff
@@ -75,7 +75,7 @@ p {
 ul,
 ol {
   font-weight: 300;
-  font-size: 2rem;
+  font-size: 1.7rem;
```

Comment:
This `font-size` is used across many places. It could be refactored to use a variable.